assimilation_code/modules/observations/default_quantities_mod.f90

```diff
@@ -167,7 +170,9 @@
 !   QTY_LIVE_STEM_CARBON
 !   QTY_LIVE_STEM_NITROGEN
 !   QTY_MEAN_SOURCE
+!   QTY_MESOZOOPLANKTON_CARBON
 !   QTY_MICROWAVE_BRIGHT_TEMP
+!   QTY_MICROZOOPLANKTON_CARBON
```

Comment:
default_quantities_mod is missing many of the new ocean qtys (ocean_quantities_mod.f90)